templates.ts

```diff
@@ -64,6 +64,9 @@ export function generateRendererFile(viewHash: string, viewFunction: string, dec
     } else {
         initBody = initBodyWithoutDecoder(viewHash, viewFunction);
     }
+    let newLinesStr = newLines !== undefined ? (newLines.toString().charAt(0).toUpperCase() + newLines.toString().slice(1)) : "True";
+    let indentStr = indent !== undefined ? indent.toString() : "4";

```

Comment:
no need for toString here

Reply:
hmm but typescript throws an error when i remove toString?